package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "dependencies": {
     "@types/hast": "^2.0.0",
-    "hast-util-raw": "^7.2.0",
+    "hast-util-raw": "^8.0.0",
     "unified": "^10.0.0"
   },
   "devDependencies": {
```

Comment:
`@types/node` has to be installed, otherwise we might get an old one from some dependency

Reply:
Good point, added to dev dependencies